app/vminsert/relabel/relabel.go

```diff
@@ -49,24 +56,68 @@ func Init() {
 	configTimestamp.Set(fasttime.UnixTimestamp())
 
 	go func() {
-		for range sighupCh {
-			configReloads.Inc()
```

Comment:
Previously, this metric was incremented whenever a signal was received. It reflected a reload *attempt*, not an actual reload. Now, the counter increments only when the config is successfully reloaded.

I had a similar discussion with @hagen1778 while working on https://github.com/VictoriaMetrics/VictoriaMetrics/pull/9575. Back then, I suggested moving the counter to reflect actual reloads rather than attempts, but the decision was to increment it before attempting a reload.

And, another question, should the counter be incremented if the `relabelConfigCheckInterval` timer is set?

Reply:
with a small interval defined at `relabelConfigCheckInterval` this counter will be incremented to frequently, but making increment only for reloads looks strange, makes sense to move this counter before `pcsNew, err := loadRelabelConfig()`